### ISSUE TITLE
Misses a section where the spies are made aware of each other

### ISSUE THREAD

**Issue by jancborchardt** [WesselWessels/resistance#1]
How I remember the game, there is a section after the roles are distributed and before the actual rounds start where the spies need to know each other.

The [Wikipedia article on The Resistance](https://en.wikipedia.org/wiki/The_Resistance_%28game%29#Gameplay) also refers to it as »The Imperial Spies are made aware of each other without the Resistance members knowing«.

What we usually did is everyone closes their eyes, heads down, and hammers their hands on the table – then the spies (and only them) are asked to look up to know who they are.

**Comment by WesselWessels** [comment]
Yes, that is correct. I just made this for myself at first, so I knew to do the whole eyes closes ceremony. I thought about using text to speech to do the ceremony. I think I should at least add a message telling the user to do the eyes closed thing.
